src/components/FileDetail.tsx

```diff
@@ -0,0 +1,65 @@
+import { Typography } from '@material-ui/core';
+import { graphql } from 'babel-plugin-relay/macro';
+import React from 'react';
+import { useLazyLoadQuery } from 'react-relay';
+import { useParams } from 'react-router-dom';
+import KeyValueTable from './KeyValueTable';
+import TruncateText from './TruncateText';
+import { FileDetailQuery } from './__generated__/FileDetailQuery.graphql';
+
+interface FileDetailParams {
+  id: string;
+}
+
+const fileDetailQuery = graphql`
+  query FileDetailQuery($id: ID!) {
+    node(id: $id) {
+      ... on File {
+        id
+        file_name
+        file_size
+        file_url
+        md5_digest
+        meta {
+          k
+          v
+        }
+      }
+    }
+  }
+`;
+
+const FileDetail: React.FC = () => {
+  const { id } = useParams<FileDetailParams>();
+  const data = useLazyLoadQuery<FileDetailQuery>(fileDetailQuery, { id });
+
+  if (!data?.node) {
+    return (
+      <Typography variant="h5" gutterBottom>
+        File Detail: Id Not Found
+      </Typography>
+    );
+  }
+
+  return (
+    <>
+      <Typography variant="h5" gutterBottom>
+        File Detail (id: {data?.node?.id})
```

Comment:
@wooneh pls move this field after **MD5 digest**

Reply:
Do you mean like this?

![image](https://user-images.githubusercontent.com/21334474/120884179-8745d880-c635-11eb-91f5-2e52ee37f2e6.png)

Reply:
Oh blast - my bad. no I meant the download link, sorry! Next time I won't rely on comments to tag the line :)

Reply:
@wooneh I'll commit this PR as is, and we can tweak it later :)